working-groups/WORKING-GROUPS.md

```diff
@@ -112,26 +112,35 @@ conventions
 | <img width="30px" src="https://github.com/navidshaikh.png"> | Navid Shaikh | VMware | [navidshaikh](https://github.com/navidshaikh) |
 | <img width="30px" src="https://github.com/rhuss.png">       | Roland Huß   | Red Hat | [rhuss](https://github.com/rhuss)            |
 
-## Documentation
+## Documentation + User Experience
 
 Knative documentation, especially the [Docs](https://github.com/knative/docs/)
 repo.
 
-| Artifact                   | Link                                                                                                                                                                   |
-| -------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Charter                    | [Charter](./docs/CHARTER.md)                                                                                                                                           |
-| Forum                      | [knative-docs@](https://groups.google.com/forum/#!forum/knative-docs)                                                                                                  |
-| Community Meeting VC       | See the top of the [Meeting notes](https://docs.google.com/document/d/13-neXVH-1n1ELWgukSzySnWj-4X6UB2pGGws2z2NH3M/edit)                                               |
-| Community Meeting Calendar | Every Tuesday, 9:30-10:30am PST<br>[Calendar](https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com) |
-| Meeting Notes              | [Notes](https://docs.google.com/document/d/13-neXVH-1n1ELWgukSzySnWj-4X6UB2pGGws2z2NH3M/edit)                                                                          |
-| Document Folder            | [Folder](https://drive.google.com/drive/folders/1cMXOSiGo1kiWl0WXeGz8nV6esLetJS2f)                                                                                     |
-| Repos                      | [`knative/docs`](https://github.com/knative/docs), [`knative/website`](https://github.com/knative/website)                                                             |
-| Slack Channel              | [#docs](https://slack.knative.dev/messages/docs)                                                                                                                       |
-| Github Team WG leads       | [@knative/docs-wg-leads](https://github.com/orgs/knative/teams/docs-wg-leads/members)                                                                                  |
 
 
-| &nbsp;                                                     | Leads            | Company | Profile                                     |
-| ---------------------------------------------------------- | ---------------- | ------- | ------------------------------------------- |
+
```

Comment:
Shouldn't there be a single heading for "DUX", now there are two (Docs + UX and UX)

Reply:
ah, ok on it